#DPATCHLEVEL=0

2006-01-11  Alfred M. Szmidt  <ams@gnu.org>

	* Makerules.in (migs_d_SCRIPT): New variable.
	(%.migs_d): Use it.
	(migu_d_SCRIPT): New variable.
	%.migu_d): Use it.

--- Makerules.in.orig	2006-01-31 11:21:04.000000000 +0200
+++ Makerules.in	2006-02-01 23:28:54.000000000 +0200
@@ -118,13 +118,17 @@
 
 # Here is how to make those dependency files
 
+migs_d_SCRIPT = 'set -e; $(CPP) $(MIGFLAGS) -M -MG -x c $< | \
+	sed -e \'s/[^:]*:/$(@:.migs_d=_server.c) $@:/\' > $@)'
+
 %.migs_d: %.srv
-	(set -e; $(CPP) $(MIGFLAGS) -M -MG -x c $< | \
-	sed -e 's/[^:]*:/$(@:.migs_d=_server.c) $@:/' > $@)
+	$(migs_d_SCRIPT)
+
+migu_d_SCRIPT = 'set -e; $(CPP) $(MIGFLAGS) -M -MG -x c $< | \
+	sed -e \'s/[^:]*:/$(@:.migu_d=_user.c) $@:/\' > $@)'
 
 %.migu_d: %.cli
-	(set -e; $(CPP) $(MIGFLAGS) -M -MG -x c $< | \
-	sed -e 's/[^:]*:/$(@:.migu_d=_user.c) $@:/' > $@)
+	$(migu_d_SCRIPT)
 
 %.migsh_d: %.migs_d
 	sed -e 's/_server\.c /_interface.h /' -e 's/migs_d/migsh_d/' < $< > $@
